fidesctl/tests/core/test_generate_dataset.py

```diff
@@ -617,3 +637,93 @@ def test_dataset_coverage_manifest_passes_snowflake(self, test_config, tmpdir):
             url=test_config.cli.server_url,
             headers=test_config.user.request_headers,
         )
+
+
+@pytest.mark.redshift
+@pytest.mark.external
+class TestRedshift:
+    EXPECTED_COLLECTION = {
+        "public": {
+            "visit": ["email", "last_visit"],
+            "login": ["id", "customer_id", "time"],
+        }
+    }
+
+    @pytest.fixture(scope="class", autouse=True)
+    def redshift_setup(self):
```

Comment:
More of a thought than anything to do for this here (likely feeding off of the great work reducing code duplication around `get_db_collections_and_fields`!) but it would be nice if we could refactor some of these tests to be more reusable as well instead of having duplication across tests per connector

Reply:
I think there is definitely some room for parameterization here, now that they all function much more similarly

@earmenda do you think a test refactor makes sense here or would you rather wait and see what happens with mongo?

Reply:
I was a little stuck on this but it definitely makes sense to condense this. It might be simple to do this as a parameterized class but I still need to figure out now to make it work with the custom `external` mark i had added. I'm okay doing this with the mongo work to keep things moving along.